tests/test_demo.py

```diff
@@ -2,4 +2,4 @@
 
 import subprocess
 
-subprocess.call("proselint --debug >/dev/null", shell=True)
+subprocess.call(["proselint", "--debug", ">/dev/null"])
```

Comment:
The same thing applies here (though you've added `shell=True` back on `master`).